src/features/device-heartbeat/index.ts

```diff
@@ -189,10 +189,9 @@ export class DeviceOnlineStateManager extends events.EventEmitter {
 			return;
 		}
 
-		// initialise the RedisSMQ object using our Redis client...
 		this.rsmq = new RedisSMQ({
-			host: REDIS.general.host,
-			port: REDIS.general.port,
+			// TODO: This cannot currently support a redis cluster
```

Comment:
What is 'this' refering to? logging device-heartbet to a cluster?

Reply:
RSMQ, basically we cannot initialize this with a cluster as it will fail